.env

```diff
@@ -192,3 +196,40 @@ TELEMETRY_EVENT_PDATA_ID=dev.uci.samagra
 
 POSTHOG_TELEMETRY_APIKEY=posthogkey
 POSTHOG_TELEMETRY_URL=https://app.posthog.com/
+
+# uci-admin
+# These variables are needed at build time
+NG_APP_url=''
+NG_APP_nl_url=''
+NG_APP_nl_login_url=''
+NG_APP_nl_login_token=''
+NG_APP_nl_application_id=''
+NG_APP_blobUrl=''
+NG_APP_botPhoneNumber=''
+NG_APP_adapterId=''
+NG_APP_broadcastAdapterId=''
+NG_APP_userId=''
+NG_APP_orgId=''
+NG_APP_token=''
+NG_APP_conversationAdapterId=''
+
+#Doubtnut config
+DOUBTNUT_BASE_URL=https://micro.doubtnut.com/secure/ask/tara
```

Comment:
https://github.com/samagra-comms/transformer/blob/master/src/main/resources/application.properties - this has to be fixed first at the transformer service. We can add a comment stating the variables are optional and will be fixed.

Reply:
Hey @ChakshuGautam 

I have added default values in transfromer for Doubtnut and NL env. variables, so that we can remove from docker-deploy .env
and I have created a PR for it can you please review it : https://github.com/samagra-comms/transformer/pull/64